File: src/entities/open-position.ts
import { SeniorityEnum } from "./enums/seniority.enum";
import { Community } from "./community";

export class OpenPosition {
  id: number;
  title: string;
  seniority: SeniorityEnum;
  studio: string;
  community: Community;
  jobDescription: string;
  priority: boolean;
}

File: src/Domain/Domain.Model/OpenPosition.cs
// <copyright file="OpenPosition.cs" company="Softvision">
// Copyright (c) Softvision. All rights reserved.
// </copyright>

namespace Domain.Model
{
    using Core;
    using Domain.Model.Enum;

    public class OpenPosition : Entity<int>
    {
        public string Title { get; set; }

        public Seniority Seniority { get; set; }

        public string Studio { get; set; }

        public Community Community { get; set; }

        public bool Priority { get; set; }

        public string JobDescription { get; set; }
    }
}
